src/web/wms/src/test/java/org/geoserver/wms/web/data/StyleEditPageTest.java

```diff
@@ -232,14 +231,6 @@ public void testInsertImage() throws Exception {
 
         tester.executeAjaxEvent("dialog:dialog:content:form:userPanel:image", "change");
         tester.assertVisible("dialog:dialog:content:form:userPanel:display");
-        assertTrue(
-                ((ResourceReference)
-                                ((org.apache.wicket.markup.html.image.Image)
-                                                tester.getComponentFromLastRenderedPage(
-                                                        "dialog:dialog:content:form:userPanel:display"))
-                                        .getDefaultModelObject())
-                        .getName()
-                        .equals("somepicture.png"));
```

Comment:
I assume this was removed because wicket is a pain and no longer exposes the image name when using the new stream-based implementation?
Have you verified manually that the image preview still shows up?

While I would prefer a similar check here in place of this one, I understand that it might not be feasible to get that working with wicket, so won't insist upon it.